Packages/com.unity.inputsystem/InputSystem/Actions/InputActionAsset.cs

```diff
@@ -583,6 +570,23 @@ public InputAction FindAction(string actionNameOrId, bool throwIfNotFound = fals
                         break;
                     }
                 }
+
+                // It's just a simple action name. Return either first enabled action or, if
+                // none are enabled, first action with the given name.
```

Comment:
Obviously, I have no idea how this function is supposed to work, but the comment looks inaccurate to me. It appears to return either the first enabled action, or the first action that matched id (can be a disabled one in this case), or can be both disabled and with the wrong id (when the return clause wasn't hit at line 583). This looks a bit weird to my uneducated glance.

Reply:
The comment predates the PR. I didn't pay too much attention since internal and didn't change that part (only moved)

Reply:
Yeah the Find has this weird quirk that it prioritises enabled actions which I find odd.

Reply:
Attempted to clarify the previous comment here: https://github.com/Unity-Technologies/InputSystem/pull/2160/commits/19887019366d8b35e47244de8b8fecf2abcf7945, it seems inline with xmldoc above?